src/api/status/public/style.css

```diff
@@ -1,5 +1,8 @@
-#results {
-  max-width: 600px;
+#sidenav-collapse-main > ul > li > a.active {
+  background-color: #121d59;
+}
+#sidenav-collapse-main.collapse.navbar-collapse.w-auto.max-height-vh-100 {
```

Comment:
The id alone should be enough to target the sidebar, but I don't see any difference with or without this rule.

Reply:
This one makes difference, Before I add the style the lower part of the navbar was blocked. 
![image](https://user-images.githubusercontent.com/50640730/142476583-f122a946-ef42-4ad5-8444-2c3046c54b5d.png)
So, I added the style to display the full navbar.
And for the ID, I also tried with ID but it doesn't work because the default CSS selector from the material dashboard was too narrow with selector. So, I narrow it down more to apply the style for the navbar.

Reply:
Can you double-check this again? Doesn't happen on my end. 

!important should be better used here. With the current selector, we remove one class in the sidebar and it'll break.

Reply:
Yap, I checked it again and it happen on my side. And !important also... but it didn't work.